include/freetds/utils/test_base.h

```diff
@@ -0,0 +1,64 @@
+#ifndef _tdsguard_afBM6E9n8CuIFSBHNNblq5
+#define _tdsguard_afBM6E9n8CuIFSBHNNblq5
+
+/*
+ * Base header for FreeTDS unit tests, even those just covering helpers
+ * from the utils and replacements trees.  Should be included first
+ * (possibly via a common.h) to be certain of preceding <assert.h>.
+ */
+
+/* Ensure assert is always active. */
+#undef NDEBUG
+#ifdef assert
+#  error "Include test_base.h (or common.h) earlier"
```

Comment:
This test turns out to be too strong for some tests. The problem is that some tests include, at the beginning, a C file and that's supposed to be the first thing they do (all some linking trick). Would be enough to check that NDEBUG is not defined instead ?